workers/javascript/tests/index.test.ts

```diff
@@ -1,102 +1,276 @@
-import { MockDBController, releasesJsQueryMock } from './mongodb.mock';
-import { beautifyUserAgent } from '../src/utils';
-/**
- * Mock db controller
- */
-jest.mock('../../../lib/db/controller', () => ({
-  DatabaseController: MockDBController,
-}));
-
-import JavascriptEventWorker from '../src'; // eslint-disable-line
-import { JavaScriptEventWorkerTask } from '../types/javascript-event-worker-task'; // eslint-disable-line
-import '../../../env-test'; // eslint-disable-line
-
-const addons = {
-  window: {
-    innerHeight: 1337,
-    innerWidth: 960,
-  },
-  userAgent: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:80.0) Gecko/20100101 Firefox/80.0',
-  url: 'https://error.hawk.so',
-};
-
-/**
- * Testing Event
- */
-const testEventData = {
-  projectId: '5d2213fa0f290c0144a69c46',
-  catcherType: 'errors/javascript',
-  payload: {
-    title: 'TestError: Everything is fine.',
-    release: null,
-    timestamp: 1564948772936,
-    backtrace: null,
-    get: null,
-    addons,
-    user: null,
-    context: null,
-  },
-} as JavaScriptEventWorkerTask;
-
-const testEventDataWithRelease = {
-  projectId: '5d2213fa0f290c0144a69c46',
-  catcherType: 'errors/javascript',
-  payload: {
-    title: 'TestError: Everything is fine.',
-    release: '3fa0f290c014',
-    timestamp: 1564948772936,
-    backtrace: [
-      {
-        file: 'file:///test.js',
-        line: 22,
-        column: 13,
+import JavascriptEventWorker from '../src';
+import '../../../env-test';
+import { JavaScriptEventWorkerTask } from '../types/javascript-event-worker-task';
+import { Db, MongoClient, ObjectId } from 'mongodb';
+import * as WorkerNames from '../../../lib/workerNames';
+import { ReleaseDBScheme } from 'hawk.types';
+
+describe('JavaScript event worker', () => {
+  let connection: MongoClient;
+  let db: Db;
+
+  /**
+   * Returns new ObjectId as string
+   */
+  const objectIdAsString = (): string => {
+    return (new ObjectId()).toHexString();
+  };
+
+  /**
+   * Parsed user agent for comparing
+   */
+  const beautifiedUserAgent = {
+    os: 'Windows',
+    osVersion: '10.0.0',
+    browser: 'Firefox',
+    browserVersion: '80.0.0',
+  };
+
+  /**
+   * Creates event object for JS worker
+   *
+   * @param withUserAgent - is event with user agent
+   * @param withBacktrace - is event with backtrace
+   */
+  const createEventMock = ({ withUserAgent, withBacktrace }: {withUserAgent?: boolean, withBacktrace?: boolean}): JavaScriptEventWorkerTask => {
+    return {
+      catcherType: 'errors/javascript',
+      projectId: objectIdAsString(),
+      payload: {
+        title: 'Mocked event for JS event worker',
+        type: 'Error',
+        timestamp: Date.now(),
+        release: '3fa0f290c014',
+        addons: {
+          window: {
+            innerHeight: 1337,
+            innerWidth: 960,
+          },
+          userAgent: withUserAgent && 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:80.0) Gecko/20100101 Firefox/80.0',
```

Comment:
I'd suggest moving it to the `originalUserAgent` variable and place it near the `beautifiedUserAgend`